internal/api/phone.go

```diff
@@ -89,14 +89,7 @@ func (a *API) sendPhoneConfirmation(r *http.Request, tx *storage.Connection, use
 		if err != nil {
 			return "", internalServerError("error generating otp").WithInternalError(err)
 		}
-
-		message, err := generateSMSFromTemplate(config.Sms.SMSTemplate, otp)
-		if err != nil {
-			return "", err
-		}
-
-		// Hook should only be called if SMS autoconfirm is disabled
-		if !config.Sms.Autoconfirm && config.Hook.SendSMS.Enabled {
+		if config.Hook.SendSMS.Enabled {
```

Comment:
no need to check for `config.Sms.Autoconfirm` here because we only call `sendPhoneConfirmation` if autoconfirm is disabled